config/config.php

```diff
@@ -226,6 +226,33 @@
             ],
         ],
 
+        /*
+         * Validators configurations. Corresponding validator will be created and
+         * will be injectable into components.
+         */
+        'validators' => [
+            'complexSetupValidator' => [
+                /*
+                 * First possible complex setup indicator is a big amount of available sessions lengths.
+                 * If the amount > 2, complex setup validation wouldn't pass.
+                 */
+                [
+                    'field' => 'sessions',
```

Comment:
Here's what's confusing:
If you read the rule, it appears that there's some list called "sessions", and that list's length must be between 0 and 2.

Reply:
Apparently, that's what it is. I read it right.

Reply:
I read it right. That's exactly what it is. Nothing confusnig.